docs/internals/contributing/triaging-tickets.txt

```diff
@@ -414,8 +414,8 @@ Bisecting a regression
 
 .. highlight:: console
 
-A regression is a bug that's present in some newer version of Django but not in
-an older one. An extremely helpful piece of information is the commit that
+A regression is a bug that's present in some newer versions of Django but not
+in older ones. An extremely helpful piece of information is the commit that
```

Comment:
I don't see a benefit to changing this.

Reply:
Me too.

Reply:
```suggestion
A Regression is a bug that's present in a newer version of Django but not
in older ones. An extremely helpful piece of information is the commit that
```

Reply:
```suggestion
A regression is a bug that's present in a newer version of Django but not
in older ones. An extremely helpful piece of information is the commit that
```

Reply:
Still don't see much of a benefit, a regression can go unfixed for several versions, so "some newer versions" isn't wrong, and I would follow Mariusz's instincts here about minimizing work for translators and just fixing clear typos.